src/PokemonPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Link, useParams } from 'react-router-dom'
+import Loader from './Loader'
+import { useApi } from './useApi'
+import ErrorMessage from './ErrorMessage'
+
+const PokemonPage = ({ pokemons }) => {
+  const { name } = useParams()
+  const { data: pokemon, error, isLoading } = useApi(`https://pokeapi.co/api/v2/pokemon/${name}`)
+
+  if (isLoading) {
+    return <Loader />
+  }
+  if (error) {
+    return <ErrorMessage error={error} />
+  }
+
+  const { type } = pokemon.types.find((type) => type.slot === 1)
+  const stats = pokemon.stats.map((stat) => ({
+    name: stat.stat.name.replace('-', ' '),
+    value: stat.base_stat
+  })).reverse()
+  const normalAbility = pokemon.abilities.find((ability) => !ability.is_hidden)
+  const hiddenAbility = pokemon.abilities.find((ability) => ability.is_hidden === true)
+  const previous = pokemons.find(({ id }) => id === pokemon.id - 1)
+  const next = pokemons.find(({ id }) => id === pokemon.id + 1)
```

Comment:
I feel like these should come in as props. A page for a single Pokemon should really not consume ALL the pokemon.